src/promptflow-evals/promptflow/evals/synthetic/simulator/simulator/simulator.py

```diff
@@ -0,0 +1,735 @@
+                return Simulator._from_openai_chat_completions(fn, simulator_connection, ai_client, ml_client, **kwargs)
+        return Simulator(
+            simulator_connection=simulator_connection,
+            ai_client=ai_client,
+            ml_client=ml_client,
+            simulate_callback=fn,
+        )
+
+    @staticmethod
+    def _from_openai_chat_completions(
+        fn: Callable[[Any], dict], simulator_connection=None, ai_client=None, ml_client=None, **kwargs
+    ):
+        return Simulator(
+            simulator_connection=simulator_connection,
+            ai_client=ai_client,
+            ml_client=ml_client,
+            simulate_callback=Simulator._wrap_openai_chat_completion(fn, **kwargs),
+        )
+
+    @staticmethod
+    def _wrap_openai_chat_completion(fn, **kwargs):
+        async def callback(chat_protocol_message):
+            response = await fn(messages=chat_protocol_message["messages"], **kwargs)
+
+            message = response.choices[0].message
+
+            formatted_response = {"role": message.role, "content": message.content}
+
+            chat_protocol_message["messages"].append(formatted_response)
+
+            return chat_protocol_message
+
+        return callback
+
+    @staticmethod
+    def from_pf_path(  # pylint: disable=unused-argument
+        pf_path: str,
+        simulator_connection: "AzureOpenAIModelConfiguration" = None,  # type: ignore[name-defined]
+        ai_client: "AIClient" = None,  # type: ignore[name-defined]
+        ml_client: "MLClient" = None,  # type: ignore[name-defined]
+        **kwargs,
+    ):
+        """
+        Creates an instance of Simulator from a specified promptflow path.
+
+        :param pf_path: The path to the promptflow folder
+        :type pf_path: str
+        :param simulator_connection: Configuration for the connection to the simulation service, if any.
+        :type simulator_connection: AzureOpenAIModelConfiguration, optional
+        :param ai_client: The AI client to be used for interacting with AI services.
+        :type ai_client: AIClient, optional
+        :keyword ml_client: An instance of MLClient for interacting with the AI service. Defaults to None.
+        :paramtype ml_client: Optional[MLClient]
+        :return: An instance of the class configured with the specified policy file,
+                 simulation connection, and AI client.
+        :rtype: The class which this static method is part of.
+        :return: An instance of simulator configured with the specified function, simulation connection, and AI client.
+        :rtype: Simulator
+        :raises ValueError: If both `simulator_connection` and `ai_client` are not provided (i.e., both are None).
+
+        Any additional keyword arguments (`**kwargs`) will be passed to the underlying configuration
+        or initialization methods.
+        """
+        try:
+            from promptflow import load_flow

```

Comment:
Please look at pf package. This has been updated to a new package

Reply:
updated to `from promptflow.client import load_flow`